pyproject.toml

```diff
@@ -4,7 +4,17 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "godice"
-authors = [{name = "Particula", email = "iavtomator@gmail.com"}]
+authors = [
```

Comment:
@Geek5510 please update/extend authors list as needed. I recognize my work was based on knowledge of others about device API